src/winetricks

```diff
@@ -15936,7 +15935,9 @@ load_wmp10()
 
     # Crashes on exit, but otherwise ok; see https://bugs.winehq.org/show_bug.cgi?id=12633
     w_try_cd "$W_CACHE/$W_PACKAGE"
-    w_try "$WINE" MP10Setup.exe $W_UNATTENDED_SLASH_Q
+    w_try cabextract -d "$W_TMP" ./MP10Setup.exe
+    w_try cd "$W_TMP"
+    w_try_cd "$WINE" setup_wm.exe $W_UNATTENDED_SLASH_CAP_QUIET
```

Comment:
I think you misunderstood. It should be:

```
w_try_cd "$W_TMP"
w_try "$WINE" setup_wm.exe $W_UNATTENDED_SLASH_CAP_QUIET
```

Ideally, squashed into one commit.

Reply:
What a blooper... Sorry, I`ll redo all over